app/src/main/java/ch/epfl/sdp/login/RegisterFormActivity.java

```diff
@@ -71,20 +79,20 @@ private void register(String username, String email, String password) {
                     if (!addUserRes.isSuccessful()) {
                         Toast.makeText(RegisterFormActivity.this, addUserRes.getException().getMessage(), Toast.LENGTH_LONG).show();
                     } else {
-                        RegisterFormActivity.this.startActivity(new Intent(RegisterFormActivity.this, MainActivity.class));
-                        RegisterFormActivity.this.finish();
+                        startActivity(new Intent(RegisterFormActivity.this, MainMenuActivity.class));
+                        finish();
                     }
                 });
             }
         });
     }
 
-    public void backBtn_OnClick(View view) {
+    private void backBtn_OnClick(View view) {
         startActivity(new Intent(RegisterFormActivity.this, LoginFormActivity.class));
```

Comment:
Is it not better to use findbyId :) to be consitent

Reply:
I use findById if you look 10 lines up...